hr_timesheet_overtime_rate_begin_end/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Make the two modules compatible.
```

Comment:
i know it’s obvious if we look into the manifest, but stating the module names here would be useful for readers, and also in the manifest, as the description is shown in odoo without listing the dependencies.